### PR TITLE
updates to reflect current business model at Agilion

### PR DESCRIPTION
**Pull request by adambouchard**
Updates to reflect

1. A transition to value-based fees for client work
2. Simplification of processes to workflow guidelines
3. Clarify vacation policy with simpler CTO time that is easier to track & schedule

### CODE REVIEW

**Comment by alanpeabody** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -225,9 +204,27 @@ General grounds for dismissal are as follows:
...
+| Memorial Day | Last Monday in May |
+| Independence Day | July 4 |
+| Labor Day | First Monday in September |
+| [Programmer's Day](http://en.wikipedia.org/wiki/Programmers'_Day) | 256th day of the year (September 13; September 12 in leap years) |
+| Thanksgiving Day | Fourth Thursday in November |
+| Christmas Day | December 25 |
 
-###Why we have processes
+
+We expect employees to take upto 20 days (four work weeks) of combined time off each year for either personal days or vacation.
```
Comment (patrickberkeley) [inline README.md]: I believe `upto` should be `up to`. I might shorten this to:

```
We expect employees to take 20 business days off each year.
```

File: README.md [hunk 2]
```diff
@@ -225,9 +204,27 @@ General grounds for dismissal are as follows:
...
+
+Time off requests are made through a google form to help other memebers in the team plan accordingly.
```
Comment (patrickberkeley) [inline README.md]: I would change this to:

```
Time off requests must be submitted with reasonable lead time. The shorter the lead time, the less likely the request will be granted due to scheduling needs with our clients.
```

**Comment by patrickberkeley** [comment]
Left a few nits, but LGTM
